### PR TITLE
#83 Fix set filters message i18n

### PR DESCRIPTION
**Pull request by oubchid**
Demo: https://streamable.com/jch4y5

### CODE REVIEW

File: src/linagora.esn.unifiedinbox/app/services/mailboxes-filter/mailboxes-filter-service.js [hunk 1]
```diff
@@ -200,7 +200,10 @@ angular.module('linagora.esn.unifiedinbox')
     }
 
     function _setFiltersOnServer(filters) {
-      return asyncJmapAction({ success: 'Filters set', failure: 'Error setting filters' }, function(client) {
+      return asyncJmapAction({
+        success: esnI18nService.translate('Filters set'),
+        failure: 'Error setting filters'
```
Comment (rouazana) [inline src/linagora.esn.unifiedinbox/app/services/mailboxes-filter/mailboxes-filter-service.js:205]: why the failure is not translated?

Comment (MichaelBailly) [reply src/linagora.esn.unifiedinbox/app/services/mailboxes-filter/mailboxes-filter-service.js:205]: @oubchid : please also add translation for "Error setting filters"

**Review by rouazana — APPROVED** [review]

File: src/linagora.esn.unifiedinbox/app/services/mailboxes-filter/mailboxes-filter-service.js (hunk 1 above)
Comment (oubchid) [reply src/linagora.esn.unifiedinbox/app/services/mailboxes-filter/mailboxes-filter-service.js]: done @MichaelBailly @rouazana

File: src/linagora.esn.unifiedinbox/i18n/fr.json [hunk 2]
```diff
@@ -76,6 +77,7 @@
   "File %s ignored as its size exceeds the %s limit": "Le fichier %s ignore que sa taille dépasse la limite %s",
   "Filter name": "Nom du filtre",
   "Filters": "Filtres",
+  "Filters set": "Filtres créés",
```
Comment (rezk2ll) [inline src/linagora.esn.unifiedinbox/i18n/fr.json:80]: `Error setting filters` and `Filters set` translations do not have the same meaning.
for one `set` means created and for the other means `configured`.

so either we use
```json
 "Filters set": "Filtres configured"
```
 or 
```json
"Error setting filters": "Erreur lors de la creation des filtres"
```

or something else completely different, something that better explains what `set` means.

**Review by rezk2ll — DISMISSED** [review]

File: src/linagora.esn.unifiedinbox/i18n/vi.json [hunk 3]
```diff
@@ -76,6 +77,7 @@
   "File %s ignored as its size exceeds the %s limit": "Bỏ qua file %s bởi dung lượng lớn hơn giới hạn: %s",
   "Filter name": "Tên bộ lọc",
   "Filters": "Bộ lọc",
+  "Filters set": "Bộ lọc đã tạo",
```
Comment (tuanlc) [inline src/linagora.esn.unifiedinbox/i18n/vi.json:80]:
```suggestion
  "Filters set": "Bộ lọc đã được thiết lập",
```

File: src/linagora.esn.unifiedinbox/i18n/vi.json [hunk 4]
```diff
@@ -63,6 +63,7 @@
   "Enable forwarding": "Chuyển tiếp",
   "Enable JMAP sending": "Gửi qua JMAP",
   "End date must be greater than start date": "Ngày kết thúc phải ở sau ngày bắt đầu",
+  "Error setting filters": "Lỗi khi đặt bộ lọc",
```
Comment (tuanlc) [inline src/linagora.esn.unifiedinbox/i18n/vi.json:66]:
```suggestion
  "Error setting filters": "Lỗi khi thiết lập bộ lọc",
```

**Review by rezk2ll — APPROVED** [review]

**Review by rouazana — DISMISSED** [review]

**Review by rezk2ll — APPROVED** [review]

**Review by fabienmoyon — APPROVED** [review]

**Pull request merged**